app/models/user.rb

```diff
@@ -0,0 +1,13 @@
+class User < ActiveRecord::Base
+	has_secure_password
+	has_many :tweets
+
+	def slug
+		self.username.downcase.gsub(" ","-")
```

Comment:
This is interesting...wouldn't it be possible to have multiple users with the same slug? For example '@john smith' and '@john-smith'.